client/src/Components/Header/index.jsx

```diff
@@ -1,9 +1,130 @@
-import React from 'react';
-
-export default function Header() {
-  return (
-    <>
-      <h1>header</h1>
-    </>
-  );
+import React, { Component } from 'react';
+import { NavLink, Link } from 'react-router-dom';
+import { Navbar, Nav, Col, Dropdown } from 'react-bootstrap';
+import PropTypes from 'prop-types';
+import Notification from './Notification';
+import './style.css';
+
+class Header extends Component {
+  state = {
+    userInfo: {
+      fullName: 'Ayman AlQoqa',
+      username: 'Ayman321396',
+      avatar:
+        'https://m.media-amazon.com/images/M/MV5BMTcxOTk4NzkwOV5BMl5BanBnXkFtZTcwMDE3MTUzNA@@._V1_.jpg',
+    },
+  };
+
+  componentDidMount() {
+    // const userInfo = localStorage.getItem('userInfo');
+    // this.setState({ userInfo });
+  }
+
+  render() {
+    const { islogged } = this.props;
+    // default values
+    const { fullName, username, avatar } = this.state.userInfo;
```

Comment:
```javascript
  const {
      userInfo: { fullName, username, avatar },
    } = this.state;

```
I think will solve the eslint problem

Reply:
we will change it when we use socket
thanks